### DISCUSSION TITLE
Workflow diagram

### DISCUSSION THREAD

**Discussion by Jamespcman** [inventree/InvenTree#8293] [Q&A]
Hello, is there a workflow diagram located anywhere on the website? I’m doing my best to understand how the ordering, Sales process works with parts and inventory.

**Comment by SchrodingersGat** [comment]
An overall workflow diagram does not exist on our [documentation site](https://docs.inventree.org/en/stable/). The documentation is a work in progress (contributions are very welcome)!

**Reply by Jamespcman** [reply]
Thank you so much for the reply. You guys are doing a great job with this software. Once I navigate through everything I’ll try to put something together.
